sample-android/src/main/java/care/data4life/sdk/helpers/stu3/AttachmentBuilderPatched.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2021 D4L data4life gGmbH / All rights reserved.
+ *
+ * D4L owns all legal rights, title and interest in and to the Software Development Kit ("SDK"),
+ * including any intellectual property rights that subsist in the SDK.
+ *
+ * The SDK and its documentation may be accessed and used for viewing/review purposes only.
+ * Any usage of the SDK for other purposes, including usage for the development of
+ * applications/third-party applications shall require the conclusion of a license agreement
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+
+package care.data4life.sdk.helpers.stu3
+
+import care.data4life.fhir.stu3.model.Attachment
+import care.data4life.fhir.stu3.model.FhirDateTime
+import care.data4life.sdk.config.DataRestriction
+import care.data4life.sdk.lang.DataRestrictionException
+import care.data4life.sdk.util.Base64
+import care.data4life.sdk.util.HashUtil
+import care.data4life.sdk.util.MimeType
+
+object AttachmentBuilderPatched {
```

Comment:
This is due to the fact, that the FHIR-helpers are still one version behind right?

Reply:
They are on an incompatible version of the utils, next time such refactorings happen they need to be marked as breaking.